agimus_controller/tests/test_robot_models.py

```diff
@@ -0,0 +1,164 @@
+from copy import deepcopy
+from os.path import dirname
+import unittest
+
+import example_robot_data as robex
+import numpy as np
+import pinocchio as pin
+
+from agimus_controller.factory.robot_model import RobotModelParameters, RobotModels
+
+
+class TestRobotModelParameters(unittest.TestCase):
+    def test_valid_initialization(self):
+        """Test that the dataclass initializes correctly with valid input."""
+
+        robot = robex.load("panda")
+        urdf_path = robot.urdf
+        srdf_path = robot.urdf.replace("urdf", "srdf")
+        urdf_meshes_dir = dirname(dirname(robot.urdf))
+        free_flyer = False
+        q0 = np.zeros(robot.model.nq)
+        params = RobotModelParameters(
+            q0=q0,
+            free_flyer=free_flyer,
+            locked_joint_names=["panda_joint1", "panda_joint2"],
```

Comment:
Can you make `["panda_joint1", "panda_joint2"]` into a variable and do a deepcopy of it for the test?